build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -145,37 +145,37 @@
       </_DownloadAndExtractItem>
 
       <_DownloadAndExtractItem Include="DownloadedSharedFrameworkInstallerFile"
-                               Condition="!Exists('$(DownloadedSharedFrameworkInstallerFile)') And '$(InstallerExtension)' != ''">
+                               Condition="'$(SkipBuildingInstallers)' != 'true' And !Exists('$(DownloadedSharedFrameworkInstallerFile)') And '$(InstallerExtension)' != ''">
```

Comment:
I will defer to @piotrpMSFT and @livarcocc, but IMO I've seen a mix of both styles all over that I don't think one way or the other is "the preferred style".